docs/InstallEbpf.md

```diff
@@ -8,20 +8,31 @@ install or update the eBPF installation in the VM.
 
 Do the following from within the VM:
 
-1. Download latest VC++ Redist package from [this location](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).
-2. Install the redist package downloaded in the above step.
-3. Download the `.msi` file from the [latest release on GitHub](https://github.com/microsoft/ebpf-for-windows/releases).
-4. Execute the `.msi` file you downloaded.
-
-The following components are shown in the MSI to select from:
-
-* **Runtime**: this is the base eBPF runtime, and is required by the other components.  If you select only this
-  component, only [native code generation](NativeCodeGeneration.md) is enabled.
-* **JIT**: this adds support for JIT-compiled eBPF programs and (in a Debug build only) interpreted eBPF programs.
-* **Development**: this adds headers and libraries used for development.  If you only want to use eBPF for development
-  rather than running programs, you can [use the NuGet package](GettingStarted.md#using-ebpf-in-development)
-  instead of the MSI.
-* **Testing**: this adds tests for the eBPF runtime for use by eBPF runtime developers.
+1. Download the `ebpf-for-windows_installer.msi` file from the [latest release on GitHub](https://github.com/microsoft/ebpf-for-windows/releases).
+1. Execute the `ebpf-for-windows_installer.msi` file you downloaded.
+1. After accepting the License and selecting the desired installation folder (default will be "`C:\Program Files\eBPFforWindows`"), the following components will be selectable from the *Installation Wizard*:
+
+    * **Runtime Components** (mandatory): this feature adds the eBPF runtime and core components, which are also required by the other components. If you select only this
+      feature, only [native code generation](NativeCodeGeneration.md) is enabled.
+    * **Development** (optional): this feature adds headers and libraries used for development. If you only want to use eBPF for development
```

Comment:
What happened to the "JIT" feature?

Reply:
@dthaler Previously, the "JIT" feature was just present in the documentation, but not in the actual MSI... therefore the documentation was documenting an inexistent MSI feature.

If there "is" a "JIT" feature that was supposed to be there, then let's sync offline and happy to package it.

Reply:
See https://github.com/microsoft/ebpf-for-windows/blob/main/tools/setup/CMakeLists.txt#L90
It's basically ebpfsvc.exe.

Reply:
Ok to deal with that in a separate PR though and track it as part of issue #1883.

Reply:
From what I can see in the previous MSI database (and the resulting GUI), the service was installed as part of the "Runtime components":

```xml
<Component Id="CM_CP_Runtime.ebpfsvc.exe" Guid="{990D477D-5AE7-56FF-B296-5D269A9473AA}" Win64="yes">
                        <File Id="CM_FP_Runtime.ebpfsvc.exe" Name="ebpfsvc.exe" KeyPath="yes" DiskId="1" Source="SourceDir\File\CM_FP_Runtime.ebpfsvc.exe" />
</Component>
```

![image](https://user-images.githubusercontent.com/22826130/213006464-a3a4d0ff-d378-4361-8d5b-07e3e0be87f4.png)


If we do need to split the service as a separate "JIT" feature, then I'd propose to just do it now OR, I can just specify more clearly in the documentation that the JIT component is among the "Runtime components".

Reply:
if it's easy to do now, feel free since I think it will help with #1883.

Reply:
Implemented partitioning for JIT, and available for Debug builds only.